### PR TITLE
added troubleshooting guide, improved report a bug guide

### PR DESCRIPTION
**Pull request by fabienlem**
Added troubleshooting guide
Improved report a bug guide

Goals:

- Have a generic documentation "First aid kit" in order to help user to eliminate the most trivial bug causes.
- Have documentation which help users to qualify the bugs "Report a Bug" and "Bug qualification".
- Have generic articles for known issues.

### CODE REVIEW

File: contributing/report_bug.rst [hunk 1]
```diff
@@ -1,21 +1,32 @@
 Report a Bug
 ============
 
+Prerequisite
+------------
+
+.. warning::
+
+    As a prerequisite, be sure you completed all the steps described in our :doc:`/troubleshooting/first_aid_kit/index`.
```
Comment (LaureBro) [inline contributing/report_bug.rst]: - make sure instead of be sure
- remove the space before the ":"

File: contributing/report_bug.rst [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
 Community Edition
 -----------------
 
-* Check if the issue is reproducible on http://demo.akeneo.com/user/login
+* Check if the issue is reproducible on http://demo.akeneo.com
 * Check if this issue is already known: https://github.com/akeneo/pim-community-dev/issues
 * Clone https://github.com/akeneo/pim-community-dev and checkout the branch of the version you want to test
 * Reproduce the bug
-* Create a new github issue that:
+* Gather as much information as possible by following our :doc:`/troubleshooting/bug_qualification/index` guide
```
Comment (LaureBro) [inline contributing/report_bug.rst]: - Gathers (add "s")
- remove space before the ":"

File: contributing/report_bug.rst [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
+* Create a new github issue which:
```
Comment (LaureBro) [inline contributing/report_bug.rst]: Creates (add the "s")

**Comment by LaureBro** [comment]
GTM 👍

File: troubleshooting/bug_qualification/index.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
+Bug qualification
+=================
+
+Bug qualification is a very important step. The more relevant information you will provide, the more valuable help you will get.
+
+.. warning::
+
+    * If you are a **Enterprise Edition Client**, be aware that the bug qualification is required when you submit a ticket on our **helpdesk**.
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: If you are an

File: troubleshooting/bug_qualification/index.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+------------
+
+.. warning::
+
+    As a prerequisite, be sure you completed all the steps described in our :doc:`/troubleshooting/first_aid_kit/index`.
+
+Frontend bugs
+-------------
+
+Frontend bugs are javascript or displaying related issues.
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: display

File: troubleshooting/bug_qualification/index.rst [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+What to do?
+
+* Use your browser buit-in developer tools to get more precise error messages and to debug javascript code step by step
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: step by step.

File: troubleshooting/bug_qualification/index.rst [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+* A frontend error can occur because the backend encountered an error, use the network tab of the developer tools to see if it's the case or not.
+* The PIM can send many type of AJAX requests (GET, POST, PUT, DELETE...), make sure the server is configured to accept all of them.
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: many types of AJAX

File: troubleshooting/bug_qualification/index.rst [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+Backend bugs
+------------
+
+Backend bugs are issues which occur on the server side (PHP/Symfony code, database...).
+
+What to do?
+
+* Use the Symfony buit-in developer tools (the debug toolbar and the profiler)
+* Use the Symfony and PIM built-in debug commands (to see them, use the command ``php app/console`` without any argument)
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: any arguments

File: troubleshooting/bug_qualification/index.rst [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+* Check the to do list of the "Backend bugs" section
+
+Performances issues
+-------------------
+
+When the PIM is sloooowwwww, it's probably a performance issue.
+
+What to do?
+
+* Use the Symfony's profiler timeline tab to identify what part of the execution take most of the time
```
Comment (MarieMinasyan) [inline troubleshooting/bug_qualification/index.rst]: takes

File: troubleshooting/common_issues/index.rst [hunk 10]
```diff
@@ -0,0 +1,68 @@
+Common issues
+=============
+
+.. tip::
+
+    Feel free to improve this documentation by clicking on the "Edit on GitHub" link on the top right corner. You can also have a look to our ":doc:`/contributing/documentation`" guide.
```
Comment (MarieMinasyan) [inline troubleshooting/common_issues/index.rst]: have a look at

File: troubleshooting/common_issues/index.rst [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+I have an error when I upload large files
+-----------------------------------------
+
+If you upload a file in an image or a file attribute, check the "max file size" parameter inside the attribute itself.
+
+In order to achieve this, follow these steps:
+
+* Go on **Settings > Attributes**, then open your image or file attribute type.
```
Comment (MarieMinasyan) [inline troubleshooting/common_issues/index.rst]: Go to

File: troubleshooting/common_issues/index.rst [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+* Increase the value of the "**max file size**" parameter or set it to blank for an unlimited file size.
+
+If you perform a file upload in the asset module, be aware there is no limitation inside Akeneo PIM.
+
+In addition, your local installation can set limitations in several places:
+
+* Check the "**upload_max_filesize**" and "**post_max_size**" paramters inside your php.ini configuration file.
```
Comment (MarieMinasyan) [inline troubleshooting/common_issues/index.rst]: parameters

File: troubleshooting/common_issues/index.rst [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+To fix this issue, just run the following commands:
+
+.. code-block:: txt
+
+    cd /path/to/your/pim/
+    rm -rf web/translations/*
+    php app/console oro:translation:dump
+
+Some Javascript, CSS or medias files are not taken into account
```
Comment (MarieMinasyan) [inline troubleshooting/common_issues/index.rst]: media files

File: troubleshooting/first_aid_kit/index.rst [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+
+.. image:: /troubleshooting/unicorn-akeneo_128.png
+
+
+Step 1: are you sure that it's a bug?
+-------------------------------------
+
+Sometimes, it's quite clear that the behaviour you experience is a bug. But it is not always the case. Some behaviours can occur because you missed something (for instance a permissions misconfiguration), and the PIM just works as designed.
+
+If you have doubts, please have a look to the following resources. If not, go to the next step.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: have a look at

File: troubleshooting/first_aid_kit/index.rst [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+* The PIM user guides: https://www.akeneo.com/user-guide/
+
+.. tip::
+
+    User guides can be accessed at any moment from the **(?)** icon on the top right corner of the PIM.
+
+Step 2: is your computer ready to use the PIM?
+----------------------------------------------
+
+There is some client side requirements to use the PIM. Please check that everything is OK by looking at :doc:`/reference/technical_information/client_side_compatibilities`.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: There are

File: troubleshooting/first_aid_kit/index.rst [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+* The PIM commands are well run in prod mode instead of dev mode (See: https://symfony.com/doc/2.7/configuration/environments.html#selecting-the-environment-for-console-commands)
+* Your crontab scheduled PIM commands are not launched too frequently
+
+Step 4: what about your infrastructure?
+---------------------------------------
+
+Depending on your infrastructure, you may need to check some points:
+
+* Make sure you are not experiencing latency issues on your local network.
+* Make sure you don't have an issue with one of your network equipment for instance with a proxy server.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: equipments

File: troubleshooting/first_aid_kit/index.rst [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+
+
+Step 5: is your PIM up to date?
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: up-to-date

File: troubleshooting/first_aid_kit/index.rst [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+-------------------------------
+
+Make sure that you applied the latest patches available for your PIM version.
+
+.. tip::
+
+    The :doc:`/developer_guide/migration/index` documentation explains how to update your PIM to the most recent minor version.
+
+Step 6: are your additional bundles up to date?
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: up-to-date

File: troubleshooting/first_aid_kit/index.rst [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+-----------------------------------------------
+
+As for the PIM it's important to check that the additional bundles you use (for instance: EnhancedConnectorBundle, CustomEntityBundle...) are up to date.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: up-to-date

File: troubleshooting/first_aid_kit/index.rst [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+
+If you need assistance on this purpose, please refer to each bundle's documentation.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: remove "on this purpose"

File: troubleshooting/first_aid_kit/index.rst [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+
+Step 7: are your assets properly deployed?
+------------------------------------------
+
+.. tip::
+
+    This step is recommended if you encounter user interface issues like javascript errors or displaying problems.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: display

File: troubleshooting/first_aid_kit/index.rst [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+
+From a technical point of view, assets are all the javascript, css and medias files used by the PIM itself, by its dependencies, and eventually by your custom developments.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: media

File: troubleshooting/first_aid_kit/index.rst [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+One last thing, clear your browser's cache:
+
+* If you use Chrome: https://support.google.com/accounts/answer/32050?hl=en
+* If you use Firefox: https://support.mozilla.org/en-US/kb/how-clear-firefox-cache
+* If you use another browser, be aware this one isn't officially supported.
+
+Step 8: did you clear the cache?
+--------------------------------
+
+Clear the PIM cache (as known as the name "Symfony cache") by running the following commands:
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: (also known as "Symfony cache")

File: troubleshooting/bug_qualification/index.rst [hunk 24]
```diff
@@ -0,0 +1,67 @@
+Bug qualification
+=================
+
+Bug qualification is a very important step. The more relevant information you will provide, the more valuable help you will get.
+
+.. warning::
+
+    * If you are a **Enterprise Edition Client**, be aware that the bug qualification is required when you submit a ticket on our **helpdesk**.
+    * This step is important as well if you are a **Community Edition user** and if you decide to share your issue on our **Slack user group**, our **GitHub repository**, our **forum** or on our **Knowledge Base**
+
```
Comment (damien-carcel) [inline troubleshooting/bug_qualification/index.rst]: 👍

File: contributing/report_bug.rst [hunk 25]
```diff
@@ -1,21 +1,32 @@
...
```
Comment (rybus) [inline contributing/report_bug.rst]: GitHub

File: troubleshooting/first_aid_kit/index.rst [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+
+.. code-block:: txt
+
+    cd /path/to/your/pim/
+    php app/console cache:clear --env=prod
+
+Step 9: did you consider the volumetrics of your catalog?
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: volume

File: troubleshooting/first_aid_kit/index.rst [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+---------------------------------------------------------
+
+.. tip::
+
+    This step is recommended if you encounter performances issues.
+
+Each catalog is unique. How many channels, locales, attributes, families, categories, products and users do you have? This values may had increase drastically since you began to use your PIM.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: may have increased... began using

File: troubleshooting/bug_qualification/index.rst (hunk 4 above)
Comment (rybus) [inline troubleshooting/bug_qualification/index.rst]: an

File: troubleshooting/first_aid_kit/index.rst [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+
+Most of this information can be found on the PIM system information screen (under System > System information).
+
+Once the calculations made, please have a look to our :doc:`/reference/scalability_guide/index` and :doc:`/reference/performances_guide/index`.
```
Comment (MarieMinasyan) [inline troubleshooting/first_aid_kit/index.rst]: at our :doc:

File: troubleshooting/bug_qualification/index.rst (hunk 5 above)
Comment (rybus) [inline troubleshooting/bug_qualification/index.rst]: JavaScript

File: troubleshooting/bug_qualification/index.rst [hunk 29]
```diff
@@ -0,0 +1,67 @@
...
+.. warning::
+
+    As a prerequisite, be sure you completed all the steps described in our :doc:`/troubleshooting/first_aid_kit/index`.
+
+Frontend bugs
+-------------
+
+Frontend bugs are javascript or displaying related issues.
+
+What to do?
```
Comment (damien-carcel) [inline troubleshooting/bug_qualification/index.rst]: In bold

File: troubleshooting/bug_qualification/index.rst [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+* Use your browser buit-in developer tools to get more precise error messages and to debug javascript code step by step
+* A frontend error can occur because the backend encountered an error, use the network tab of the developer tools to see if it's the case or not.
+* The PIM can send many type of AJAX requests (GET, POST, PUT, DELETE...), make sure the server is configured to accept all of them.
+
+Backend bugs
+------------
+
+Backend bugs are issues which occur on the server side (PHP/Symfony code, database...).
+
+What to do?
```
Comment (damien-carcel) [inline troubleshooting/bug_qualification/index.rst]: In bold too

File: troubleshooting/bug_qualification/index.rst [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+* Use the Symfony and PIM built-in debug commands (to see them, use the command ``php app/console`` without any argument)
+* Use the tools of your integrated development environment to debug the code step by step
+* Analyze the error messages provided in the Symfony's log files ``/path/to/your/pim/app/logs/prod.log`` or ``/path/to/your/pim/app/logs/dev.log``
+
+Tasks bugs
+----------
+
+Tasks bugs are issues which occur when running the PIM jobs, like mass edit, import, export... (The way to qualify could be a bit different than pure backend bugs).
+
+What to do?
```
Comment (damien-carcel) [inline troubleshooting/bug_qualification/index.rst]: Again

File: troubleshooting/bug_qualification/index.rst [hunk 32]
```diff
@@ -0,0 +1,67 @@
...
+* Analyze the error messages provided in the following file ``/path/to/your/pim/app/logs/batch_execute.log``
+* Launch the command out of the PIM with your the tools of your integrated development environment ``php app/console akeneo:batch:job <batch_name> <additional_params>``
+* Check the to do list of the "Backend bugs" section
+
+Performances issues
+-------------------
+
+When the PIM is sloooowwwww, it's probably a performance issue.
+
+What to do?
```
Comment (damien-carcel) [inline troubleshooting/bug_qualification/index.rst]: And again

File: troubleshooting/first_aid_kit/index.rst (hunk 27 above)
Comment (damien-carcel) [inline troubleshooting/first_aid_kit/index.rst]: These values may have

File: troubleshooting/bug_qualification/index.rst [hunk 33]
```diff
@@ -0,0 +1,67 @@
...
+* The PIM can send many type of AJAX requests (GET, POST, PUT, DELETE...), make sure the server is configured to accept all of them.
+
+Backend bugs
+------------
+
+Backend bugs are issues which occur on the server side (PHP/Symfony code, database...).
+
+What to do?
+
+* Use the Symfony buit-in developer tools (the debug toolbar and the profiler)
```
Comment (anaelChardan) [inline troubleshooting/bug_qualification/index.rst]: (the debug toolbar and the profiler).

File: troubleshooting/bug_qualification/index.rst [hunk 34]
```diff
@@ -0,0 +1,67 @@
...
+* Use the Symfony and PIM built-in debug commands (to see them, use the command ``php app/console`` without any argument)
+* Use the tools of your integrated development environment to debug the code step by step
```
Comment (anaelChardan) [inline troubleshooting/bug_qualification/index.rst]: step by step.

File: troubleshooting/bug_qualification/index.rst [hunk 35]
```diff
@@ -0,0 +1,67 @@
...
+Tasks bugs
+----------
+
+Tasks bugs are issues which occur when running the PIM jobs, like mass edit, import, export... (The way to qualify could be a bit different than pure backend bugs).
+
+What to do?
+
+* Analyze the error messages provided in the following file ``/path/to/your/pim/app/logs/batch_execute.log``
+* Launch the command out of the PIM with your the tools of your integrated development environment ``php app/console akeneo:batch:job <batch_name> <additional_params>``
+* Check the to do list of the "Backend bugs" section
```
Comment (anaelChardan) [inline troubleshooting/bug_qualification/index.rst]: section.

File: troubleshooting/first_aid_kit/index.rst [hunk 36]
```diff
@@ -0,0 +1,142 @@
...
+    php app/console pim:installer:check-requirements
+
+.. note::
+
+    All our system requirements can also be found in this documentation:
+    :doc:`/developer_guide/installation/system_requirements/system_requirements`.
+
+In addition, check that:
+
+* The Xdebug PHP extension is well deactivated
```
Comment (anaelChardan) [inline troubleshooting/first_aid_kit/index.rst]: deactivated.

File: troubleshooting/first_aid_kit/index.rst [hunk 37]
```diff
@@ -0,0 +1,142 @@
...
+* The PIM commands are well run in prod mode instead of dev mode (See: https://symfony.com/doc/2.7/configuration/environments.html#selecting-the-environment-for-console-commands)
+* Your crontab scheduled PIM commands are not launched too frequently
```
Comment (anaelChardan) [inline troubleshooting/first_aid_kit/index.rst]: frequently.

File: troubleshooting/bug_qualification/index.rst [hunk 38]
```diff
@@ -0,0 +1,67 @@
...
+
+Backend bugs are issues which occur on the server side (PHP/Symfony code, database...).
+
+What to do?
+
+* Use the Symfony buit-in developer tools (the debug toolbar and the profiler)
+* Use the Symfony and PIM built-in debug commands (to see them, use the command ``php app/console`` without any argument)
+* Use the tools of your integrated development environment to debug the code step by step
+* Analyze the error messages provided in the Symfony's log files ``/path/to/your/pim/app/logs/prod.log`` or ``/path/to/your/pim/app/logs/dev.log``
+
```
Comment (a2xchip) [inline troubleshooting/bug_qualification/index.rst]: app/logs/batch_execute.log

File: troubleshooting/bug_qualification/index.rst [hunk 39]
```diff
@@ -0,0 +1,67 @@
+Bug qualification
+=================
+
+Bug qualification is a very important step. The more relevant information you will provide, the more valuable help you will get.
+
+.. warning::
+
+    * If you are an **Enterprise Edition Client**, be aware that the bug qualification is required when you submit a ticket on our **helpdesk**.
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: link to the helpdesk?

File: troubleshooting/bug_qualification/index.rst [hunk 40]
```diff
@@ -0,0 +1,67 @@
...
+    * This step is important as well if you are a **Community Edition user** and if you decide to share your issue on our **Slack user group**, our **GitHub repository**, our **forum** or on our **Knowledge Base**
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: we could link all these no?

File: troubleshooting/bug_qualification/index.rst [hunk 41]
```diff
@@ -0,0 +1,67 @@
...
+------------
+
+Backend bugs are issues which occur on the server side (PHP/Symfony code, database...).
+
+**What to do?**
+
+* Use the Symfony buit-in developer tools (the debug toolbar and the profiler).
+* Use the Symfony and PIM built-in debug commands (to see them, use the command ``php app/console`` without any arguments).
+* Use the tools of your integrated development environment to debug the code step by step.
+* Analyze the error messages provided in the Symfony's log files ``/path/to/your/pim/app/logs/prod.log`` or ``/path/to/your/pim/app/logs/dev.log``
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: should we talk about the webserver logs?

File: troubleshooting/bug_qualification/index.rst [hunk 42]
```diff
@@ -0,0 +1,67 @@
...
+
+Performances issues
+-------------------
+
+When the PIM is sloooowwwww, it's probably a performance issue.
+
+**What to do?**
+
+* Use the Symfony's profiler timeline tab to identify what part of the execution takes most of the time.
+* Use blackfire.io in order to profile the requests which take too much time.
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: could we provide a link?

File: troubleshooting/bug_qualification/index.rst [hunk 43]
```diff
@@ -0,0 +1,67 @@
...
+* Use MySQL built in "SHOW PROCESSLIST" query to identify queries which take too long to run.
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: link to the documentation?

File: troubleshooting/bug_qualification/index.rst [hunk 44]
```diff
@@ -0,0 +1,67 @@
...
+* Use MEMINFO to analyse memory leak issues: https://github.com/BitOne/php-meminfo
```
Comment (jjanvier) [inline troubleshooting/bug_qualification/index.rst]: direct link?

File: troubleshooting/common_issues/index.rst [hunk 45]
```diff
@@ -0,0 +1,68 @@
+Common issues
+=============
+
+.. tip::
+
+    Feel free to improve this documentation by clicking on the "Edit on GitHub" link on the top right corner. You can also have a look at our ":doc:`/contributing/documentation`" guide.
```
Comment (jjanvier) [inline troubleshooting/common_issues/index.rst]: link to the button?

File: troubleshooting/common_issues/index.rst [hunk 46]
```diff
@@ -0,0 +1,68 @@
...
+.. code-block:: txt
+
+    cd /path/to/your/pim/
+    rm -rf ./web/bundles/* ./web/css/* ./web/js/*
+    php app/console pim:install:assets --env=prod
+    php app/console assets:install --symlink web
+
+Also, don't forget to clear your browser's cache:
+
+* If you use Chrome: https://support.google.com/accounts/answer/32050?hl=en
```
Comment (phaseinducer) [inline troubleshooting/common_issues/index.rst]: Are the links parsed?

File: troubleshooting/first_aid_kit/index.rst [hunk 47]
```diff
@@ -0,0 +1,142 @@
...
+.. note::
+
+    All our system requirements can also be found in this documentation:
+    :doc:`/developer_guide/installation/system_requirements/system_requirements`.
+
+In addition, check that:
+
+* The Xdebug PHP extension is well deactivated.
+* The PIM commands are well run in prod mode instead of dev mode (See: https://symfony.com/doc/2.7/configuration/environments.html#selecting-the-environment-for-console-commands)
+* Your crontab scheduled PIM commands are not launched too frequently.
```
Comment (phaseinducer) [inline troubleshooting/first_aid_kit/index.rst]: Maybe a link to https://docs.akeneo.com/latest/developer_guide/installation/installation_ce_archive.html#configuring-tasks-via-crontab would be nice

File: troubleshooting/first_aid_kit/index.rst [hunk 48]
```diff
@@ -0,0 +1,142 @@
...
+
+Step 4: what about your infrastructure?
+---------------------------------------
+
+Depending on your infrastructure, you may need to check some points:
+
+* Make sure you are not experiencing latency issues on your local network.
+* Make sure you don't have an issue with one of your network equipments for instance with a proxy server.
```
Comment (phaseinducer) [inline troubleshooting/first_aid_kit/index.rst]: comma missing after "equipments" imho

File: troubleshooting/first_aid_kit/index.rst [hunk 49]
```diff
@@ -0,0 +1,142 @@
...
+
+
+Step 5: is your PIM up-to-date?
+-------------------------------
+
+Make sure that you applied the latest patches available for your PIM version.
```
Comment (phaseinducer) [inline troubleshooting/first_aid_kit/index.rst]: Maybe we can say that the info is available in the PIM's footer?

File: troubleshooting/first_aid_kit/index.rst [hunk 50]
```diff
@@ -0,0 +1,142 @@
...
+From a technical point of view, assets are all the javascript, css and media files used by the PIM itself, by its dependencies, and eventually by your custom developments.
+
+To make sure everything is OK, you can run the following commands:
+
+.. code-block:: txt
+
+    cd /path/to/your/pim/
+    rm -rf ./web/bundles/* ./web/css/* ./web/js/*
+    php app/console pim:install:assets --env=prod
+    php app/console assets:install --symlink web
```
Comment (phaseinducer) [inline troubleshooting/first_aid_kit/index.rst]: Duplicated section here, you should use an include.

File: troubleshooting/first_aid_kit/index.rst [hunk 51]
```diff
@@ -0,0 +1,142 @@
...
+
+One last thing, clear your browser's cache:
+
+* If you use Chrome: https://support.google.com/accounts/answer/32050?hl=en
+* If you use Firefox: https://support.mozilla.org/en-US/kb/how-clear-firefox-cache
```
Comment (phaseinducer) [inline troubleshooting/first_aid_kit/index.rst]: Idem

File: troubleshooting/common_issues/index.rst (hunk 46 above)
Comment (fabienlem) [inline troubleshooting/common_issues/index.rst]: Yes, they are.

File: troubleshooting/bug_qualification/index.rst [hunk 52]
```diff
@@ -0,0 +1,67 @@
...
+Frontend bugs
+-------------
+
+Frontend bugs are JavaScript or display related issues.
+
+**What to do?**
+
+* Use your browser buit-in developer tools to get more precise error messages and to debug javascript code step by step.
+* A frontend error can occur because the backend encountered an error, use the network tab of the developer tools to see if it's the case or not.
+* The PIM can send many types of AJAX requests (GET, POST, PUT, DELETE...), make sure the server is configured to accept all of them.
```
Comment (fabienlem) [inline troubleshooting/bug_qualification/index.rst]: Like what? :)

**Pull request merged**